Revert "Update controller-start-up.md:- Corrects a few grammatical errors" (#1699)
cyberbotics/webots: docs/guide/controller-start-up.md

The .wbt file contains the name of the controller that needs to be started for each robot.
The controller name is a platform and language independent field; for example when a controller name is specified as "xyz\_controller" in the .wbt file, this does not say anything about the controller's programming language or platform.

> This is done deliberately to ensure the platform and programming language independence of .wbt files.

Reviewer: ```suggestion
This is done deliberately to ensure that the *.wbt* file is independent from the platform and programming language.
```

> So when Webots tries to start a controller it must first determine what programming language is used by this controller.

Reviewer: ```suggestion
When Webots tries to start a controller it must first determine what programming language is used by this controller.
```

Author: Thanks for your guidance and help, sir !!

> So, Webots looks in the project's "controllers" directory for a subdirectory that matches the controller name.

Reviewer: ```suggestion
So, Webots looks in the project's *controllers* directory for a subdirectory that matches the name of the controller.
```
